ament_tools/verbs/list_dependencies.py

```diff
@@ -47,6 +47,11 @@ def prepare_arguments(parser):
         help='Show only test dependencies of a given package',
     )
     parser.add_argument(
+        '--group-deps',
+        action='store_true',
+        help='Show only group dependencies of a given package',
```

Comment:
This option is good, but if I interpret the usage of it correctly, it only shows you that a given package has a group dependency and the name of the group. It does not tell you which packages in the workspace are considered a member of that group.

So, it would be good to either expand the group dependency and display them as part of this option or have an option like `list --group-of <group name>`.

Reply:
Makes sense. dc26ceb8c29cfc3dc798cfc1b73e4ec085df26e2 lists all member of the group.

Reply:
That looks good to me, 👍.